src/content/docs/errors-inbox/mobile-tab.mdx

```diff
@@ -0,0 +1,260 @@
+    * **Device type**: The model of the device on which the crash occurred.
+    * **OS version**: The operating system version running on the device.
+    * **RAM used**: The amount of memory the application was utilizing at the time of the crash.
+    * **Disk free**: The amount of available storage space on the device when the crash happened.
+    * **Attributes**: Detailed attributes associated with the crash report, offering deeper insights into the context of the crash.
+
+You can export crash details to Xcode or resymbolicate your source code, if applicable, to streamline the troubleshooting process.
+
+## Event trail [#event-trail]
+
+The event trail provides a chronological log of events leading up to a crash, aiding root cause analysis. This timeline adheres to your subscription's data retention policy and encompasses both New Relic default events and custom events you've implemented.
+
+By default, the event trail is sorted chronologically, beginning with the oldest event, which is typically the app launch. You can modify the following:
+
+* Sort: Toggle between ascending and descending order.
+* Event Filtering: Filter by event type, like `interaction`, `request`, or request error).
+* Event Details: Expand individual events to inspect their attributes, like `responseTime`,or `requestUrl` for request events.
+
+
+After you've sorted and filtered your events, you can dig a little deeper into the events that lead up to the crash by examining:
+
+* **Custom breadcrumbs**: Utilize the [Record breadcrumb SDK](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/record-breadcrumb/) to create custom `MobileBreadcrumb` events. This allows you to log specific application interactions that may be relevant to crash analysis.
+* **Handled exceptions**: Use the recordHandledException SDK methods for iOS and Android to annotate where exceptions are handled in your application. These annotations will automatically populate the crash event trail.
+
+For detailed guidance on enhancing crash event trails with custom data, see [Record breadcrumbs](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/record-breadcrumb/).
+
+To fully leverage our crash analysis tools, make sure to:
+* Use the mobile SDK to create custom `MobileBreadcrumb` or `MobileHandledException` events. 
+* Enable `MobileRequest` events for capturing network request data.
+
+<img
+    title="group crashes event trail"
+    alt="A screenshot showing the event trail for group crashes in the UI"
+    src={errorsinboxEventTrail}
+/>
+
+<figcaption>
+    INSERT_CAPTION_HERE
+</figcaption>
+
+## Mobile stack trace [#stack-trace]
+
+The stack trace for the crashing thread is displayed beneath the crash details. Each frame within the stack indicates the associated library/package name and method. The agent attempts to highlight the specific stack frame containing the programming error responsible for the crash, if detected.
```

Comment:
I'd try to avoid the use of the slash (/)

Reply:
Fixed!